__tests__/api.test.js

```diff
@@ -156,3 +154,54 @@ describe("GET/api/articles", () => {
     })
 })
 
+describe("GET /api/articles/:article_id/comments", () => {
+    it("Should return an array of comments for the given article", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toHaveLength(11)
+            body.forEach(comment => {
+                expect(comment).toHaveProperty("comment_id")
+                expect(comment).toHaveProperty("votes")
+                expect(comment).toHaveProperty("created_at")
+                expect(comment).toHaveProperty("author")
+                expect(comment).toHaveProperty("body")
+                expect(comment).toHaveProperty("article_id")
+            })
+        })
+    })
+    it("Should return an array of comments sorted by created_at, newest first", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toBeSortedBy("created_at", {descending: true})
+        })
+    })
+    it("Should return 400 - Bad request when given an invalid article-id query", () => {
+        return request(app)
+        .get("/api/articles/banana/comments")
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("Bad request")
+        })
+    })
+    it("Should return 404 - Not found when given an article-id that contains no comments", () => {
```

Comment:
Great to see you thinking about this case, if the article exists but just has no comments we'd expect a 200 with an empty comments array as this is a good request (I can see how you're handling this in your model so a hint is you might also want to use one of your model functions to check if the articleID exists)

It also will make your life much easier when you come to put this all together the frontend.